ckanext/syndicate/tasks.py

```diff
@@ -124,8 +143,17 @@ def _create_package(package):
 
     new_package_data['extras'] = filter_extras(new_package_data['extras'])
     new_package_data['resources'] = filter_resources(package['resources'])
-    new_package_data['owner_org'] = get_syndicated_organization()
-    del new_package_data['organization']
+
+    org = new_package_data.pop('organization')
+
+    try:
+        if not is_organization_preserved():
+            raise ValueError("Ignore organization")
+        org_id = replicate_remote_organization(org)
+    except:
```

Comment:
Catching any exception here could hide a potential error - we should only catch the exceptions we are expecting to see, or if we are not sure which exceptions we are receiving (because we are not sure what the CKAN API will return) we should at least log them